app/controllers/downloads_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DownloadsController < ApplicationController
+  def content
+    work_version = WorkVersion.find_by!(uuid: params[:resource_id])
+    file = work_version.file_version_memberships.find(params[:id])
+    authorize(file)
+    redirect_to file.file_resource.file_url(expires_in: ENV.fetch('DOWNLOAD_URL_TTL', 6))
```

Comment:
I'm curious if we should use `response_content_disposition: "attachment"` here; this _should_ inform the browser to initiate a download for files it knows about such as txt, pdf, jpeg?

Reply:
The pre-signed url is already doing that somehow. See below. The Amazon url (200 GET) has the disposition set to return the original filename.
![Screen Shot 2020-03-06 at 3 11 13 PM](https://user-images.githubusercontent.com/312085/76119350-3ae89480-5fbd-11ea-8185-8a5e055316a2.png)

Reply:
After a slack discussion with @srerickson, we'll stick with the inline default, but can revisit later.